frontend/public/components/utils/managed-by.tsx

```diff
@@ -28,6 +28,14 @@ const matchOwnerAndCSV = (owner: OwnerReference, csvs: K8sResourceCommon[] = [])
   return csvs.find((csv) => isOwnedByOperator(csv, owner));
 };
 
+export const findOwner = (obj: K8sResourceCommon, data: ClusterServiceVersionKind[]) => {
```

Comment:
There's no reason to lazy load the dialog if we use this helper from this module. The module won't be lazy-loaded.

Can you move this into a different file under `public/module/k8s`?